woocommerce/woocommerce-ios
# [Woo Express Sign Up] Remove option to create WPCOM account from invalid email screen

Closes: #12375 


## Description


## Testing instructions
1. Launch app logout if needed
2. Tap "Log In"
3. Enter a valid JP connected Woo store URL and continue
4. Enter an invalid email (which is not a WPCOM account email) to login
5. In the error screen validate that you don't see the "Create An Account" to create a new WPCOM account

## Screenshots


| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 13 Pro - 2024-03-28 at 11 22 47](https://github.com/woocommerce/woocommerce-ios/assets/524475/102b1e45-36c3-4327-a1c3-040f07e3e12d) | ![Simulator Screenshot - iPhone 13 Pro - 2024-03-28 at 17 45 42](https://github.com/woocommerce/woocommerce-ios/assets/524475/4998ab54-bcad-47a1-924c-99be03510aac) | 

---
